rest/resource-server/src/docs/asciidoc/components.adoc

```diff
@@ -31,6 +31,28 @@ include::{snippets}/should_document_get_components/http-response.adoc[]
 ===== Links
 include::{snippets}/should_document_get_components/links.adoc[]
 
+[[resources-components-list]]
```

Comment:
Please change this to [[resources-components-list-all-details]], as right now its redirecting to listing components doc data in rest api docs.